.env.example

```
@@ -39,4 +39,11 @@ ACCESS_KEY_ID=your_access_key_id
 BUCKET_NAME=your_bucket_name
 BASE_URL=your_base_url
-BASE_URL_IMAGES=your_base_url_for_images_NOT_END_IN_SLASH
\ No newline at end of file
+BASE_URL_IMAGES=your_base_url_for_images_NOT_END_IN_SLASH
+
+# Email Service Configuration
+EMAIL_HOST=your_email_smtp_host
+EMAIL_PORT=your_email_smtp_port
+EMAIL_USER=your_email_smtp_user
+EMAIL_PASS=your_email_smtp_password
+EMAIL_FROM=your_email_from_address # without @ | example mail.example.com NO @mail.example.com
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)